torch/utils/mem_tracker.py

```diff
@@ -0,0 +1,242 @@
+from collections import defaultdict
+from dataclasses import dataclass
+from typing import Any, Dict
+import torch
+import math
+from torch._subclasses.fake_tensor import FakeTensorMode
+from torch.utils._pytree import tree_map_only
+import torchvision
+import torch.nn as nn
+from torch.utils._python_dispatch import TorchDispatchMode
+from torch.utils.weak import WeakIdKeyDictionary
+import weakref
+from enum import Enum
+
+MB = 2**20
```

Comment:
Since this is a public namespace, we need to make private everything that we don't want to expose to the user.
The two options here are:
- Make everything that is private start with a `_`
- Add a `__all__` attribute that lists the public APIs

Reply:
Done